plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ScriptMediatorPropertiesEditionPartForm.java

```diff
@@ -550,10 +550,11 @@ public void keyPressed(KeyEvent e) {
      */
 	protected Composite createScriptBodyText(FormToolkit widgetFactory, Composite parent) {
 		Control scriptBodyLabel = createDescription(parent, EsbViewsRepository.ScriptMediator.Properties.scriptBody, EsbMessages.ScriptMediatorPropertiesEditionPart_ScriptBodyLabel);
-		scriptBody = widgetFactory.createText(parent, ""); //$NON-NLS-1$
+		scriptBody = widgetFactory.createText(parent, "", SWT.MULTI | SWT.BORDER | SWT.WRAP | SWT.V_SCROLL); //$NON-NLS-1$
```

Comment:
Remove the comment.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804